src/Keboola/StorageApi/Options/TokenUpdateOptions.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+namespace Keboola\StorageApi\Options;
+
+class TokenUpdateOptions
```

Comment:
Tahle třída vybízí k tomu udělat na ní testy. Ono je to pak otestovaný nepřímo v těch funkčních testech a možná je to dostatečný.

Každopádně by to znamenalo udělat sekci do buildu, která by spouštěla třeba jenom `tests\unitests` a spouštěla se při buildu. Nechám rozhodnutí na tobě.

Reply:
dobře .. je mi jasný, jak se se rozhodnul :)